app/components/BalancesCard/index.tsx

```diff
@@ -29,12 +30,17 @@ export const BalancesCard: FC<Props> = (props) => {
     pklima: "pKLIMA",
     sklima: "sKLIMA",
     wsklima: "wsKLIMA",
+    wsklimaUnwrapped:
+      "wsKLIMA (" +
+      t({ id: "wsklima.unwrapped.label", message: "unwrapped" }) +
+      ")",
```

Comment:
`wsKLIMA (unwrapped)` is fine, generally better for translators to have the full context

`wsklimaUnwrapped: t({ id: "wsklima.unwrapped.label", message: "wsKLIMA (unwrapped)" })
`

Make sure to run `extract-strings` again after you change it